cmake/modules/test_config_ase.cmake

```diff
@@ -32,7 +32,6 @@ set(COMMON_SRC gtmain.cpp jsonParser.cpp
   unit/gtEnumerate.cpp
   unit/gtOptionParser.cpp
   unit/gtAnyValue.cpp
-  unit/gtsysfs.cpp
   function/gtCxxEnumerate.cpp
   function/gtCxxEvents.cpp
```

Comment:
Do events work on ASE?

Reply:
Yes, if you are talking about the interrupt support. Interrupt is supported on DCP in ASe.